TEveEventDisplay/src/TEveMu2e_base_classes/TEveMu2eMainWindow.h

```diff
@@ -72,16 +72,34 @@ class TText;
 class TGSplitFrame;
 class TGLOverlayButton;
 namespace mu2e{
+
+  struct DrawOptions{
+    // data options
+    bool addCRVInfo = false;
+    bool addCosmicTracks = false;
+    bool addTracks = false;
+    bool addClusters = false; 
+    bool addComboHits = false;
+    bool addCryHits = false;
+    bool addMCTraj = false;
+    DrawOptions(){};
```

Comment:
The advice we have from the experts is that if the compiler written d'tor will do the right thing, don't write your own d'tor, even an empty one.  This case is an example of that.